src/topology/homotopy/product.lean

```diff
@@ -260,3 +271,135 @@ end
 end prod
 
 end path.homotopic
+
+
+namespace fundamental_groupoid
+
+private abbreviation π := fundamental_groupoid_functor
+universes u
+
+section pi
+
+variables {I : Type u} {X : I → Top.{u}}
+
+/--
+The projection map Π i, X i → X i induces a map π(Π i, X i) ⟶ π(X i).
+-/
+def proj (i : I) : (π.obj (Top.of (Π i, X i))).α ⥤ (π.obj (X i)).α := π.map ⟨_, continuous_apply i⟩
+
+/-- The projection map is precisely path.homotopic.proj interpreted as a functor -/
+@[simp] lemma proj_map (i : I) (x₀ x₁ : (π.obj (Top.of (Π i, X i))).α) (p : x₀ ⟶ x₁) :
+  (proj i).map p = (@path.homotopic.proj _ _ _ _ _ i p) := rfl
+
+/--
+The map taking the pi product of a family of fundamental groupoids to the fundamental
+groupoid of the pi product. This is actually an isomorphism (see `pi_fgrpd_iso_fgrpd_pi`)
+-/
+@[simps]
+def pi_fgrpd_to_fgrpd_pi :
```

Comment:
Since you are already in a namespace, I think this can just be `pi_to_pi_Top` or something like that.